### PR TITLE
[#232] French English polishing

### PR DESCRIPTION
**Pull request by ghost**
[#232]

General English and French language polish
Open to suggestions if you think my changes are inadequate

### CODE REVIEW

File: src/beta_client/src/assets/locale.json [hunk 1]
```diff
@@ -1,67 +1,67 @@
 {
     "en": [
         {
-            "question": "Hello there stranger! My name is Zeus and I'm here to assist you with your Québec tenant/landlord issues. I'll be going into beta soon, and would like to get more context about common problems. If you don't mind telling me about any tenant/landlord issues you have, we'll try to get back to you as soon as possible with some useful information.",
-            "answer": ["Okay"],
+            "question": "Hello there stranger! My name is Zeus and I'm here to assist the tenants and landlords in Québec. I'll be going into beta soon, and would like to get more context about the problems you might be facing. Tell me about your issues with your tenants or your landlord and we'll try to get back to you as soon as possible with useful information.",
```
Comment (arekmano) [inline src/beta_client/src/assets/locale.json]: tenants and landlords _of_ Quebec

Comment (arekmano) [reply src/beta_client/src/assets/locale.json]: Tell me about your _residential_ issues and _I'll_ try to get back

File: src/beta_client/src/assets/locale.json [hunk 2]
```diff
@@ -1,67 +1,67 @@
...
+            "answer": ["Understood!"],
             "val": 1
         },
         {
-            "question": "Would you like to ask me a question about tenant/landlord issue?",
-            "answer": ["Sure", "I have no question"],
+            "question": "Would you like to ask me a question regarding issues between tenants and landlord?",
```
Comment (arekmano) [inline src/beta_client/src/assets/locale.json]: Would you like to ask me a question?

We arleady state its residential stuff in previous text blob

File: src/beta_client/src/assets/locale.json [hunk 3]
```diff
@@ -1,67 +1,67 @@
...
+            "answer": ["Yes", "I have no questions"],
```
Comment (arekmano) [inline src/beta_client/src/assets/locale.json]: I have no _question_

File: src/beta_client/src/assets/locale.json [hunk 4]
```diff
@@ -1,67 +1,67 @@
...
             "answer": ["Confirm"],
             "type": "question",
             "placeholder": "ENTER YOUR QUESTION",
-            "error": "Please make sure your input is valid and not empty, thanks!",
+            "error": "Please make sure your input is valid and not empty, thank you!",
             "val": 3
         },
         {
-            "question": "Mind leaving your email so that we can let you know once our beta is live?",
+            "question": "Would you like leaving me your email so I can let you know once my beta is live?",
```
Comment (arekmano) [inline src/beta_client/src/assets/locale.json]: Would you like _to leave_ me

File: src/beta_client/src/assets/locale.json [hunk 5]
```diff
@@ -1,67 +1,67 @@
...
-            "question": "Thank you so much! I'll be in touch. Have a nice day!",
+            "question": "Thank you so much! I'll stay in touch. Have a nice day!",
             "val": 6
         }
     ],
     "fr": [
         {
-            "question": "Bonjour cher étranger! Mon nom est Zeus et je suis ici pour vous assistez avec vos problèmes entre propriétaires et locataires au Québec. Je commencerai ma phase bêta bientôt. J'ai besoin par contre d'en connaître un peu plus à égard des problèmes communs qui peuvent survenir dans ces situations. Si m'aidez ne vous ennuie pas, nous nous efforcerons de vous communiquez de l'information utile dans les plus brefs délais.",
-            "answer": ["Okay"],
+            "question": "Bonjour cher étranger! Mon nom est Zeus et je suis ici pour assister les propriétaires et les locataires au Québec. Je commencerai ma phase bêta bientôt et j'ai besoin d'en connaître un peu plus à l'égard des problèmes communs auxquels vous pourriez faire face. Parlez-moi de vos problèmes avec vos locataires ou votre propriétaire et nous nous efforcerons de vous communiquez de l'information utile dans les plus brefs délais.",
```
Comment (arekmano) [inline src/beta_client/src/assets/locale.json]: je  m'efforcerai

File: src/beta_client/src/assets/locale.json [hunk 6]
```diff
@@ -1,67 +1,67 @@
...
+            "answer": ["D'accord!"],
             "val": 1
         },
         {
-            "question": "Voulez-vous me poser une question concernant un problème de locataire/propriétaire?",
+            "question": "Voudriez-vous me poser une question concernant un problème entre locataires et propriétaire?",
```
Comment (arekmano) [inline src/beta_client/src/assets/locale.json]: No need to repeat "entre locataires et propriétaire"

**Comment by arekmano** [comment]
Left some comments

**Comment by arekmano** [comment]
Any way to chunk up some of the statements so that multiple they are sent as multiple messages, one after the other? @choitwao does the FE support this, or is it a major change

File: src/beta_client/src/assets/locale.json (hunk 3 above)
Comment (ghost) [reply src/beta_client/src/assets/locale.json]: It's either "I have no questions" or "I don't have a question".

" I have X" is acceptable when X is plural form if an "a" is not inserted to define a quantitative measure of "1".
"I have cars" vs "I have car" vs "I have a car"
"I don't have cars" vs "I don't have car" vs "I don't have a car"

File: src/beta_client/src/assets/locale.json (hunk 2 above)
Comment (ghost) [reply src/beta_client/src/assets/locale.json]: Fair.

**Comment by ghost** [comment]
Solved requests. Disagreed with one change.

**Review by arekmano — APPROVED** [review]

**Pull request merged**